sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -45,9 +45,10 @@
     private String clientSecret;
 
     /**
-     * Redirection Endpoint: Used by the authorization server to return responses containing authorization credentials
-     * to the client via the resource owner user-agent.
+     * @deprecated Now the redirect-url-template is equal to "{baseUrl}/login/oauth2/code/"
```

Comment:
maybe a little more clear what should be changed once this property is deprecated?

Reply:
@stliu Updated.